lib/inline-view.coffee

```diff
@@ -32,7 +32,7 @@ class InlineView
     # Config value if you want to limit the status bar report
     # if your cursor is in the range or error, or on the line
     limitOnErrorRange = (atom.config.get('linter.statusBar') ==
-                         'Show error if the cursor is in range')
+                        'Show error if the cursor is in range')
```

Comment:
I actually think the origin indent is correct! [See PEP8](https://www.python.org/dev/peps/pep-0008/#indentation) for a reference